inst/templates/rnaseq/02_differential_expression/DEGpattern.Rmd

```diff
@@ -18,10 +18,7 @@ output:
 params:
    deseq_obj: "DEGpattern_deseq_obj.rds"
    deseq_meta: "DEGpattern_deseq_meta.rds"
```

Comment:
this needs to be like this: 
```
"https://raw.githubusercontent.com/bcbio/bcbioR-test-data/main/rnaseq/DEGpattern/DEGpattern_deseq_meta.rds"
```

This way, users can modify this directly and you don't need to change any thing downstream